storage/backup/backup-cli/src/bin/db-backup.rs

```diff
@@ -23,7 +23,7 @@ use std::sync::Arc;
 
 #[derive(Parser)]
 #[clap(about = "Ledger backup tool.")]
-enum Command {
+pub enum Command {
```

Comment:
keep this in backup-cli so that existing cli still works. will remove and merge when db-tool is stable.